src/ProgressBar/ProgressBar.scss

```diff
@@ -1,2 +1,10 @@
 @import "variables";
 @import "~bootstrap/scss/progress";
+
+.progress { 
+    border: 1px solid $gray-500;
+}
+.progress-bar { 
+    background-color: $accent-a;
+}
```

Comment:
Please use `$progress-bar-bg` and `$progress-bar-border-width` in these definitions

Reply:
@abutterworth fixed and added a new variable $progress-bar-border-color to store the border color.